src/dftbp/dftbplus/initprogram.F90

```diff
@@ -629,7 +632,8 @@ module dftbp_dftbplus_initprogram
     real(dp), allocatable :: gcurr(:), glast(:), displ(:)
 
     !> Charge mixer
```

Comment:
```suggestion
    !> Charge mixer for real matrices
```

Reply:
Yes, thanks.

Reply:
Analogously `!> Charge mixer for complex matrices`.